prow/tide/tide.go

```diff
@@ -1610,7 +1597,7 @@ func (c *Controller) presubmitsByPull(sp *subpool) (map[int][]config.Presubmit,
 				continue
 			}
 
-			record(int(pr.Number), ps)
+			presubmits[int(pr.Number)] = append(presubmits[int(pr.Number)], ps)
```

Comment:
It is perfectly valid to append to a nil slice, the previous code was needlessly complex and annoying to read, as you had to go back up to read the inline func